src/config/media/pipewire.md

```diff
@@ -0,0 +1,87 @@
+# PipeWire
+
+To use PipeWire, install the `pipewire` package.
+
+PipeWire should be started as a user. Use the autostarting mechanism of your
+desktop environment.
+
+If using [startx](../graphical-session/xorg.md#startx), add the following line
+in your `.xinitrc` file:
+
+```
+pipewire &
+```
+
+If using [Sway](https://swaywm.org/), start it from the configuration file:
+
+```
+exec pipewire
+```
+
+## PulseAudio replacement
+
+PipeWire can start a PulseAudio server. While it's possible to modify
+`/etc/pipewire/pipewire.conf` for auto-starting the PulseAudio server, it's
+recommended to start it from the same place where you start PipeWire by
+executing `pipewire-pulse`. This keeps PipeWire's configuration file unmodified
+for smoother future upgrades.
+
+#### Example for `.xinitrc`
+
+```
+pipewire &
+pipewire-pulse &
+```
+
+#### Example for Sway
+
+```
+exec pipewire
+exec pipewire-pulse
+```
+
+After setting up `pipewire-pulse`, make sure that the PulseAudio service is
+[disabled](../services/index.md#disabling-services) and reboot.
+
+To check if the replacement is working correctly, use the following command:
+
+```
+$ pactl info
+
+...
+Server Name: PulseAudio (on PipeWire 0.3.18)
+...
+```
+
+##### Note on Bluetooth audio
+
+If having troubles with Bluetooth audio, make sure that `libspa-bluetooth`
+package is installed.
+
+## ALSA integration
+
+Install `alsa-pipewire`, then enable the PipeWire ALSA device and make it
+default:
+
+```
+# mkdir -p /etc/alsa/conf.d
+# ln -s /usr/share/alsa/alsa.conf.d/50-pipewire.conf /etc/alsa/conf.d
+# ln -s /usr/share/alsa/alsa.conf.d/99-pipewire-default.conf /etc/alsa/conf.d
+```
```

Comment:
I think this section could have a link to `alsa` docs on its configuration directories, if you can find something.